CHANGELOG

```diff
@@ -1,3 +1,9 @@
+2.0.3 2015-03-27
+- Fix `graphite-web` install when using pip and default `root_dir`
+- Restart services when changing package versions
+- Improve acceptance and integration test coverage
+- Remove support for Ruby 1.8.7 and 1.9.2
```

Comment:
Not sure. It feels a bit excessive, especially as we haven't actually broken support for those versions, we just don't intend to support them anymore. But we might do in the future and it might be useful to indicate that by a version change now. Maybe compromise on a minor version bump?